dex/farm/src/lib.rs

```diff
@@ -75,25 +68,22 @@ pub trait Farm:
             "Farming token ID cannot be farm token ID"
         );
 
-        self.state().set_if_empty(&State::Active);
+        self.state().set(&State::Inactive);
```

Comment:
This is ok, but we have to remove the check for active state from farm_token module in which the setup is made. Otherwise, we either set the state back to Active or we can't continue with the setup

Reply:
done